tests/storage/linstor/conftest.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from lib.common import retry_command
+
+GROUP_NAME = 'linstor_group'
+
+@pytest.fixture(scope='session')
+def lvm_disks(host):
+    disks = []
+    hosts = host.pool.hosts
+
+    for host in hosts:
+        local_disks = host.disks()
+        assert len(local_disks) > 1, "at least two disks are required"
+        disks.append('/dev/' + local_disks[1])
+
+    i = 0
+    for host in hosts:
+        host.ssh(['pvcreate', '-ff', '-y', disks[i]])
+        host.ssh(['vgcreate', GROUP_NAME, disks[i]])
+        i += 1
+
+    yield disks
+
+    i = 0
+    for host in hosts:
+        host.ssh(['vgremove', GROUP_NAME])
+        host.ssh(['pvremove', disks[i]])
+        i += 1
+
+@pytest.fixture(scope='session')
+def hosts_with_linstor(host):
+    packages = ['drbd', 'kmod-drbd', 'linstor-client', 'linstor-controller', 'linstor-satellite', 'python-linstor']
+    hosts = host.pool.hosts
+
+    # Configure master host.
+    host.yum_install(packages)
+    host.ssh(['systemctl', 'restart', 'linstor-satellite'])
+    host.ssh(['systemctl', 'restart', 'linstor-controller'])
+
+    # Waiting for startup...
+    retry_command(lambda: host.ssh_with_result(['linstor', 'node', 'list']).returncode)
+
+    # Configure slaves.
+    for host in hosts[1:]:
```

Comment:
are we sure `hosts[0]` is the master?

Reply:
The host param given in the function is checked here: https://github.com/xcp-ng/xcp-ng-tests/blob/4d51e1b9be7d8ef73ed52f193239de75dd22c302/conftest.py#L60

Concerning this line: https://github.com/xcp-ng/xcp-ng-tests/blob/a5432c813f2be921421845b17731f1b27d6681db/tests/storage/linstor/conftest.py#L33
We always have the master at the offset 0: https://github.com/xcp-ng/xcp-ng-tests/blob/4d51e1b9be7d8ef73ed52f193239de75dd22c302/lib/common.py#L95
hosts = host.pool.hosts
```,